YSE_App/frb_init.py

```diff
@@ -0,0 +1,28 @@
+""" Methods to init the DB """
+
+from YSE_App.models import TransientStatus
+from YSE_App.data_utils import add_or_grab_obj
+
+def init_status(user):
+
+    # Delete all existing
+    TransientStatus.objects.all().delete()
+
+    # Create the FRB ones
+    all_status = [\
+        'Unassigned', # Likely just ingested although most will be given PublicPATH
+        'PublicPATH', # Needs to be run through PATH with public data
+        'Image', # Needs deeper imaging
+        'Spectrum', # Needs spectroscopy for redshift
+        'DeepPATH', # Needs PATH run on deeper imaging
+        'PendingImage', # Pending deeper imaging with an FRBFollowUp
+        'PendingSpectrum', # Pending spectroscopy with an FRBFollowUp
```

Comment:
`PendingMask`?

Reply:
I don't think we need to distinguish here.